### PR TITLE
Quick admin csv export for footnotes adapted from source/docs csv #189

### PR DESCRIPTION
**Pull request by rlskoeser**

### CODE REVIEW

File: geniza/footnotes/admin.py [hunk 1]
```diff
@@ -356,6 +356,67 @@ def doc_relation_list(self, obj):
...
+                footnote.doc_relation,
+                footnote.notes,
+                footnote.url,
+                "\n".join(footnote.content["lines"]) if footnote.content else "",
+                f"{url_scheme}{site_domain}/admin/footnotes/footnote/{footnote.id}/change/",
+            ]
+
+    def export_to_csv(self, request, queryset=None):
+        """Stream source records as CSV"""
+        queryset = self.get_queryset(request) if queryset is None else queryset
```
Comment (thatbudakguy) [inline geniza/footnotes/admin.py:397]: does this do the same thing as `queryset = queryset or self.get_queryset(request)`?

File: geniza/footnotes/admin.py [hunk 2]
```diff
@@ -356,6 +356,67 @@ def doc_relation_list(self, obj):
...
+        return export_to_csv_response(
+            self.csv_filename(),
+            self.csv_fields,
+            self.tabulate_queryset(queryset),
+        )
+
+    export_to_csv.short_description = "Export selected sources to CSV"
```
Comment (thatbudakguy) [inline geniza/footnotes/admin.py:404]: aren't we technically exporting footnotes?

File: geniza/footnotes/admin.py [hunk 3]
```diff
@@ -356,6 +356,67 @@ def doc_relation_list(self, obj):
...
+
+    def get_urls(self):
+        """Return admin urls; adds a custom URL for exporting all sources
+        as CSV"""
```
Comment (thatbudakguy) [inline geniza/footnotes/admin.py:408]: same q about footnote vs. source

File: geniza/footnotes/admin.py [hunk 4]
```diff
@@ -356,6 +356,67 @@ def doc_relation_list(self, obj):
...
+        site_domain = Site.objects.get_current().domain.rstrip("/")
+        # qa / prod always https
+        url_scheme = "https://"
+
+        for footnote in queryset:
+            yield [
+                footnote.content_object,
+                footnote.source,
+                footnote.location,
+                footnote.doc_relation,
```
Comment (thatbudakguy) [inline geniza/footnotes/admin.py:388]: `doc_relation` actually prints something like `Edition,Translation`, right? i remember it works a little weirdly with `__str__` where it's sometimes a friendly value and sometimes just `["E", "T"]`

**Review by thatbudakguy — APPROVED** [review]

File: geniza/footnotes/admin.py (hunk 1 above)
Comment (rlskoeser) [reply geniza/footnotes/admin.py:397]: yeah, good catch — should refactor all three of these

File: geniza/footnotes/admin.py (hunk 2 above)
Comment (rlskoeser) [reply geniza/footnotes/admin.py:404]: yeah, thanks. working too quickly!

File: geniza/footnotes/admin.py (hunk 4 above)
Comment (rlskoeser) [reply geniza/footnotes/admin.py:388]: django-tabular-export converts to string for us, I'm getting the full label in the export

**Pull request merged**